packages/docs-markdown/src/test/suite/helper/metadata-completion.test.ts

```diff
@@ -65,10 +65,7 @@ export const context: ExtensionContext = {
 		clear: () => {},
 		delete: () => {}
 	},
-	extensionMode: ExtensionMode.Test,
-	storageUri: undefined,
-	globalStorageUri: undefined,
-	logUri: undefined
+	extensionMode: ExtensionMode.Test
```

Comment:
I think those properties are needed, looks like the CI is failing prepublish because these properties are missing. You might need to delete your node_modules folder and re install to get the properties to show as required.